src/components/Header/Header.tsx

```diff
@@ -0,0 +1,161 @@
+                    <svg
+                        viewBox="64 64 896 896"
+                        focusable="false"
+                        className=""
+                        data-icon="menu"
+                        width="1em"
+                        height="1em"
+                        fill="currentColor"
+                        aria-hidden="true"
+                    >
+                        <path d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"></path>
+                    </svg>
+                </button>
+            </div>
+
+            {expanded ? (
+                <ul className="w-11/12 mx-auto mt-8 block lg:hidden list-none">
+                    <NavbarLink
+                        to="/product-features"
+                        textLight={textLight}
+                        className="block my-2 py-2 border-b border-white border-opacity-10"
+                    >
+                        Product
+                    </NavbarLink>
+                    <NavbarLink
+                        to="/docs"
+                        textLight={textLight}
+                        className="block my-2 py-2 border-b border-white border-opacity-10"
+                    >
+                        Docs
+                    </NavbarLink>
+                    <NavbarLink
+                        href="https://github.com/posthog/posthog"
+                        textLight={textLight}
+                        className="block my-2 py-2 border-b border-white border-opacity-10"
+                    >
+                        Community
+                    </NavbarLink>
+                    <NavbarLink
+                        to="/handbook/company/story"
+                        textLight={textLight}
+                        className="block my-2 py-2 border-b border-white border-opacity-10"
+                    >
+                        Company
+                    </NavbarLink>
+                    <NavbarLink
+                        to="/pricing"
+                        textLight={textLight}
+                        className="block my-2 py-2 border-b border-white border-opacity-10"
+                    >
+                        Pricing
+                    </NavbarLink>
+                    <NavbarLink
+                        href="https://app.posthog.com/login"
+                        textLight={textLight}
+                        className="block my-2 py-2 border-b border-white border-opacity-10"
+                    >
+                        Login
+                    </NavbarLink>
+
+                    <PrimaryCta className="my-2 ml-4">Get Started</PrimaryCta>
+                </ul>
+            ) : null}
+        </div>
+    )
+}
+
+export default Header
```

Comment:
While we're refactoring JS -> TS, can we also move away from the default export?

Here's Marius making his case to me a while ago:
https://github.com/PostHog/posthog.com/pull/511#discussion_r503230523